gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +211,55 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def background_symlinks(self, obs_table, out_dir_background_model, indir, modeltype, filename_obs_group_table=None):
```

Comment:
@cdeil 
I add this method to associate for a observation_table the background models for each observation